alica_ros_turtlesim/CMakeLists.txt

```diff
@@ -63,4 +63,8 @@ install(DIRECTORY include/${PROJECT_NAME}/
 
 install(DIRECTORY launch
     DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+)
+
+install(DIRECTORY alica
```

Comment:
you can just add add this to above line,
`install(DIRECTORY launch alica 
DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})`

Reply:
done